spring-pulsar/src/test/java/org/springframework/pulsar/listener/DefaultPulsarMessageListenerContainerTests.java

```diff
@@ -262,6 +262,7 @@ void deadLetterPolicy() throws Exception {
 		// Normal consumer should receive 5 msg + 1 re-delivery
 		assertThat(latch.await(10, TimeUnit.SECONDS)).isTrue();
 		container.stop();
+		dlqContainer.stop();
```

Comment:
So this reminders me, in all of the consumer tests we close/stop/cleanup at the end of a **successful** test - but not during a failed one. 

I don't think we need to fix that in this PR, but do you mind adding an issue for us to cleanup properly so we don't forget to come back to this?

Reply:
https://github.com/spring-projects-experimental/spring-pulsar/issues/168